bsp_diff/common/kernel/linux-intel-lts2021/0030-add-scaling-support.patch

```diff
@@ -0,0 +1,196 @@
+From b3ac61a9a3924e935805c1197621f92e56dd8aaa Mon Sep 17 00:00:00 2001
+From: Hang Liu<hang1.liu@intel.com>
+Date: Tue, 11 Jul 2023 11:16:00 +0800
+Subject: [PATCH 1/3] add scaling support
+
+the gem buffer created by i915 in guest vm may be
+smaller than the resolution determined by the
+mode setting operation, in that case, a new framebuffer
```

Comment:
instead of , after operation may be end the sentence and start the next one as new sentence.

Reply:
ok